master/internal/config/config.go

```diff
@@ -290,6 +291,11 @@ func (t *TLSConfig) ReadCertificate() (*tls.Certificate, error) {
 	return &cert, err
 }
 
+// RBACConfig is the configuration for RBAC.
+type RBACConfig struct {
+	Enabled bool `json:"enabled"`
```

Comment:
Maybe have it as an optional explicit field on AuthZConfig?

```

type AuthZConfig struct {
   ...
   RBACUIEnabled *bool `json:"rbac_ui_enabled"`
}

func (a *AuthZConfig) IsRBACUIEnabled() bool {
  if a.RBACUIEnabled != nil {
    return a.RBACUIEnabled
  }

  return a.Type != BasicAuthZType
}
```

this leaves a flexibility of explicitly enabling it but also adds sensible default